accord-core/src/main/java/accord/local/CommandStores.java

```diff
@@ -208,10 +207,10 @@ KeyRanges currentRanges()
 
         static long keyIndex(Key key, long numShards)
         {
-            return Integer.toUnsignedLong(key.keyHash()) % numShards;
+            return Integer.toUnsignedLong(key.routingHash()) % numShards;
         }
 
-        private static long addKeyIndex(Key key, long numShards, long accumulate)
+        private static long addKeyIndex(int i, Key key, long numShards, long accumulate)
```

Comment:
`i` is dead code, is this still needed?

Reply:
I think it's probably to permit its use as a method handle